src/edgeDebugAdapter.ts

```diff
@@ -24,17 +25,24 @@ export class EdgeDebugAdapter extends ChromeDebugAdapter {
 
         // Check that debug adpater executable exists
         if (!fs.existsSync(adapterExePath)) {
-            if (utils.getPlatform() == utils.Platform.Windows) {
+            if (utils.getPlatform() == utils.Platform.Windows && os.release() >= '10.0.18362') {
```

Comment:
You really want to check each component of the version separately. There's probably a library that can parse the numbers between the periods and do the comparison properly.  This will work but it doesn't reflect the best practice when dealing with version numbers.

Reply:
I don't follow - this just checks the OS release version not every component and it's similar to what we already do in office-addin-debugging to check whether to enable loopback settings

Reply:
You shouldn't check versions using a string.  "8.1.12345" is greater than "10.0. 18362"

Reply:
Good point